lib/mongodb.js

```diff
@@ -57,6 +57,11 @@ function generateMongoDBURL(options) {
   options.port = options.port || 27017;
   options.database = options.database || options.db || 'test';
   var username = options.username || options.user;
+  var portUrl = ''
```

Comment:
missing the ending `;`

Reply:
added the colon and linter is now passing